package.json

```diff
@@ -78,16 +78,13 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
-    "fbjs": "^0.8.16",
```

Comment:
Only used in docs.  `dependencies` should only include deps used in `src`.  All others are only needed for our builds and are therefore `devDependencies`.